src/vs/editor/common/config/editorOptions.ts

```diff
@@ -563,10 +563,10 @@ export interface IEditorOptions {
 	 */
 	occurrencesHighlight?: boolean;
```

Comment:
I wonder if we want to extend the original setting to `true | false | 'singleFile' | 'multiFile'`, I personally would prefer a single setting if possible.

Reply:
I agree. One setting is simpler than two. Word based suggestions should probably follow the lead

Reply:
Merged both settings and went with a settings migration for both to keep things simple.